files/tasks/packit-service-requirements.yaml

```diff
@@ -17,6 +17,8 @@
   args:
     chdir: "{{ reverse_dir }}"
   become: true
+  environment:
+    SOURCE_BRANCH: "{{ zuul.branch }}"
```

Comment:
Can you please answer some questions since I am not much aware of those changes:

- What will be installed from this `SOURCE_BRANCH`, all the dependencies?
- What will be the content of the `zuul.branch`? The branch with the PR?
- What if the `zuul.branch` does not exist in that project?

Thank you and sorry if it is too lazy from me..;)

Reply:
What will be installed from this SOURCE_BRANCH, all the dependencies?
 
yes
   
    What will be the content of the zuul.branch? The branch with the PR?

master branch will be used
    
    What if the zuul.branch does not exist in that project?

I thought it will be used via zuul, if not I can rework it:
* add a default value
* consume env variable
but will need to see specific scenarios to be able to choose the best approach

Reply:
Thanks for the answers, the third question does not make much sence now. I thought that `zuul.branch` is the PR branch name of the current project.